### ISSUE TITLE
Assert failure(PID 3300 [0x00000ce4], Thread: 15916 [0x3e2c]): Consistency check failed: ERROR: you didn't pass CONTEXT_EXCEPTION_REQUEST into GetThreadContext

### ISSUE THREAD

**Issue by gkhanna79** [dotnet/coreclr#8091]
17 P1 tests fail on this

**Comment by BruceForstall** [comment]
I see the following failures with this assert:
```
baseservices\threading\generics\threadstart\GThread02\GThread02.cmd
baseservices\threading\generics\threadstart\GThread11\GThread11.cmd
baseservices\threading\generics\threadstart\GThread12\GThread12.cmd
baseservices\threading\generics\threadstart\GThread15\GThread15.cmd
baseservices\threading\generics\threadstart\GThread17\GThread17.cmd
baseservices\threading\generics\threadstart\GThread22\GThread22.cmd
baseservices\threading\interlocked\compareexchange\compareexchangeobject\compareexchangeobject.cmd
baseservices\threading\interlocked\compareexchange\CompareExchangeTString\CompareExchangeTString.cmd
baseservices\threading\interlocked\exchange\exchangefloat\exchangefloat.cmd
GC\Features\HeapExpansion\plug\plug.cmd
GC\Scenarios\THDChaos\thdchaos\thdchaos.cmd
```

**Comment by AntonLapounov** [comment]
According to Jan Kotas, this is an OS issue that was fixed in RS3.

**Comment by jkotas** [comment]
Fixed by #12825
